<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="7" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="lbFileName2">
        <property name="text">
         <string>File name</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>File 1</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbFileName1">
        <property name="text">
         <string>File name</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="lbImageSize2">
        <property name="text">
         <string>Image size</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lbImageSize1">
        <property name="text">
         <string>Image size</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>File 2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>30</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Open...</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" rowspan="2">
       <widget class="QLabel" name="lbImage1">
        <property name="text">
         <string>[img]</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="13" column="1">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Open...</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0" rowspan="2">
       <widget class="QLabel" name="lbImage2">
        <property name="text">
         <string>[img]</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="1">
         <widget class="QComboBox" name="cbAlgorithm">
          <item>
           <property name="text">
            <string>aHash</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>pHash</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>dHash</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="4">
         <widget class="QSlider" name="tbMatch">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>95</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Match %</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Algorithm</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QLCDNumber" name="lcdNumber">
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="intValue" stdset="0">
           <number>95</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="resultTable">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string>File #1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>File #2</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Match</string>
         </property>
         <property name="textAlignment">
          <set>AlignHCenter|AlignVCenter|AlignCenter</set>
         </property>
         <property name="background">
          <color>
           <red>255</red>
           <green>239</green>
           <blue>239</blue>
          </color>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="infoLabel">
        <property name="text">
         <string>Ready...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAbort">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;Abort</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>963</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Scan_directory"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Scan_directory"/>
   <addaction name="separator"/>
   <addaction name="action_Exit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Scan_directory">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/toolbar/Search-Images-icon.png</normaloff>:/images/toolbar/Search-Images-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Scan directory...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/toolbar/Log-Out-icon.png</normaloff>:/images/toolbar/Log-Out-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Scan_directory</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>startScanDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbMatch</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>721</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>811</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startScanDirectory()</slot>
 </slots>
</ui>
